docs/how_to/single_node.md

```diff
@@ -159,3 +119,7 @@ curl -sSf -X POST http://localhost:9003/query -d '{
   "exhaustive": true
 }
 ```
+
+## Use integrations
+
+Checkout the integrations documentation to integrate with our software, such as [Prometheus as a long term storage remote read/write endpoint](../integrations/prometheus.md).
```

Comment:
Maybe just this?
`[Prometheus as a long term storage remote read/write endpoint](../integrations/prometheus.md)`

Reply:
Sure thing, done.